Makefile

```diff
@@ -102,6 +109,10 @@ USER_VARS += QEMUPATH
 USER_VARS += SHAREDIR
 USER_VARS += SHIMPATH
 USER_VARS += SYSCONFDIR
+USER_VARS += PAUSEDIR
+USER_VARS += PAUSEBIN
```

Comment:
I don't think we need to add `PAUSEDIR` and `PAUSEBIN` to `USER_VARS` as I don't think they'd realistically change, and even if they did, that would be driven by us, not by a user.